### ISSUE TITLE
Using different gateway based on dwell

### ISSUE THREAD

**Issue by brammahh** [gotthardp/lorawan-server#462]
When two gateways, connected to the same application server, are within range of each other, the web interface shows the packets are received on both gateways. When an uplink message is sent, the application server will choose to transmit on the gateway with the best RSSI.

When the dwell time of the gateway with the best RSSI is violated (i.e. higher than 1% in Europe), the application server keeps sending new uplinks via that same gateway. If a LoRa node is within reach of multiple gateways, wouldn't it make sense to use the second gateway when the first one has a violated dwell time?

**Example**:

`Gateway 1: RSSI = -25, Dwell = 0.8%`
`Gateway 2: RRSI = -40, Dwell = 0.3%`

After sending a few messages in a short timeframe, the dwell time reaches the legal limit:
`Gateway 1: Dwell = 1.4%`
`Gateway 2: Dwell = 0.3%`

What I'm proposing is to consider dwell time when deciding which gateway to use for uplink transmission. 
When both are below 1%, it makes sense to use `gateway 1` because of the RSSI. When `gateway 1`'s dwell is above 1% however, it would make sense to use `gateway 2`, staying within legal boundaries.

Do you see this as a feature worth adding in?

**Comment by gotthardp** [comment]
Yes, this would be a useful feature. The request for it is even already listed as issue #273.
